core-bundle/src/Resources/config/services.yml

```diff
@@ -1023,3 +1023,7 @@ services:
             version: 4.13
             message: Using the "%service_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "Contao\CoreBundle\String\SimpleTokenParser" instead.
         public: true
+
+    Contao\CoreBundle\Monolog\SystemLogger:
```

Comment:
According to #3686 this should be a service id + an additional alias.

Reply:
Yes, good point. I guess the ID would be `contao.monolog.system_logger`, then.

I'll wait until #3686 is merged and rebase this pull request on top of it.